Cubical/Cohomology/EilenbergMacLane/Base.agda

```diff
@@ -297,6 +341,23 @@ snd (coHomHom n f) =
   makeIsGroupHom (ST.elim2 (λ _ _ → isOfHLevelPath 2 squash₂ _ _)
    λ f g → refl)
 
+coHomEquiv : ∀ {ℓ''} {A : Type ℓ} {B : Type ℓ'} {G : AbGroup ℓ''}
+            (n : ℕ)
+         → (Iso A B)
+         → AbGroupEquiv (coHomGr n G B) (coHomGr n G A)
```

Comment:
Why is A and B swapped in the conclusion?

Reply:
wanted to stay true to the fact that coHom is contravariant